setup.py

```diff
@@ -67,7 +67,7 @@
                        optional=True)
     ],
     cmdclass={'build_ext': cpp_extension.BuildExtension},
-    install_requires=['torch>=1.6.0', 'blist', 'numpy>=1.19.1'],
+    install_requires=['torch>=1.6.0', 'blist', 'numpy>=1.19.1', 'boltons'],
```

Comment:
Is there a particular min version for boltons that is required? This is mostly reflecting from our numpy version mismatch discussion a while ago.

Reply:
Not that I'm aware of / have tried tracking down.